src/gluonts/ev/__init__.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from .metrics import (
+    mean_absolute_label,
+    sum_absolute_label,
+    SumAbsoluteError,
+    MeanSquaredError,
+    SumQuantileLoss,
+    Coverage,
+    MeanAbsolutePercentageError,
+    SymmetricMeanAbsolutePercentageError,
+    MeanScaledIntervalScore,
+    MeanAbsoluteScaledError,
+    NormalizedDeviation,
+    RootMeanSquaredError,
+    NormalizedRootMeanSquaredError,
+    WeightedSumQuantileLoss,
+)
+from .aggregations import Aggregation, Sum, Mean
+from .api import (
```

Comment:
I think this can just be evlautor?

Reply:
Pretty much, yes. I'd also like to move the Metric protocol to metrics.py but then, there's a circular import for `api.py` and `metrics.py`. Do you have an idea how to resolve this?